### PR TITLE
Added JUnit tests HussainsTest3.java

### PR DESCRIPTION
**Pull request by HussainArif12**
These tests have all assertTrue, assertFalse and other required changes from the JUnit library.

### CODE REVIEW

**Comment by HussainArif12** [comment]
Sir this is now one single commit

**Comment by SachinRammoorthy** [comment]
@HussainArif12 I think you should rename the PR to something more related to the topic, like _Added JUnit Test Cases_, for example.

**Comment by HussainArif12** [comment]
So, should I do it again?

**Comment by HussainArif12** [comment]
Okay mentor, the title has been changed. Please check my submit on GCI

File: src/test/java/org/jbossoutreach/HussainsTest3.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    public void testArrayEquals()
+    {
+        int[] arrayVals = {a, b};
+        int[] arrayValsExpected = {10, 7};
+        String[] arrayStrs = {s1, s2};
+        String[] arrayStrsExpected = {"SWASTIK", "DAS"};
+        Assert.assertArrayEquals(arrayVals, arrayValsExpected);
+        Assert.assertArrayEquals(arrayStrs, arrayStrsExpected);
+    }
+}
```
Comment (shubhamvashisht) [inline src/test/java/org/jbossoutreach/HussainsTest3.java]: Add a blank newline.

**Comment by HussainArif12** [comment]
Added the newline

**Comment by shubhamvashisht** [comment]
@HussainArif12 Travis build failed. Please check in logs and fix.

**Comment by HussainArif12** [comment]
org.jbossoutreach.HussainsTest3 > testArrayEquals FAILED
    org.junit.internal.ArrayComparisonFailure at HussainsTest3.java:61
        Caused by: org.junit.ComparisonFailure at HussainsTest3.java:61
How do I fix ?

File: src/test/java/org/jbossoutreach/HussainsTest3.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+package org.jbossoutreach;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class HussainsTest3
+{
+
+    private int a = 10;
+    private int b = 7;
+    private String s1 = "Hussain";
```
Comment (shubhamvashisht) [inline src/test/java/org/jbossoutreach/HussainsTest3.java]: String s1 and s2 are not equal to the strings in array arrayStrsExpected, Thats why your test failed. So fix it.

**Comment by HussainArif12** [comment]
I'm making another pull request.

**Comment by shubhamvashisht** [comment]
@HussainArif12 No need to make a new PR, This can be fixed.

**Pull request closed without merging**